README.md

```diff
@@ -51,14 +51,14 @@ Please refer to the [technical documentation](https://opendistro.github.io/for-e
 
 * Install OpenSearch
 
-* Install the opendistro-elasticsearch-security plugin for your Elasticsearch version 6.5.4, e.g.:
+* Install the opensearch-security plugin for your OpenSearch version 1.0.0-beta1, e.g.:
 
 ```
 <OpenSearch directory>/bin/opensearch-plugin install \
-  -b com.amazon.opendistroforelasticsearch:opendistro_security:0.8.0.0
+  -b org.opensearch:opensearch-security:1.0.0.0-beta1
```

Comment:
Please do not change you name space of `opendistroforelasticsearch`. 
That would be changed after beta1 release. This has lot more involved changes which is why we want to do it later. Probably you have to update your build.gradle file, since you already moved it to `org.opensearch`
```suggestion
  -b com.amazon.opendistroforelasticsearch:opensearch-security:1.0.0.0-beta1
```

Reply:
Done. I've changed the `group` in  `build.gradle` and `groupId` in `pom.xml` back to `com.amazon.opendistroforelasticsearch`